akka-stream-tests/src/test/java/akka/stream/javadsl/GraphDslTest.java

```diff
@@ -225,10 +224,13 @@ public void beAbleToConstructClosedGraphFromList() throws Exception {
   @Test
   public void canUseMapMaterializedValueOnGraphs() {
     Graph<SourceShape<Object>, NotUsed> srcGraph = Source.empty();
-    Graph<SourceShape<Object>, Pair<NotUsed, NotUsed>> mappedMatValueSrcGraph =
-        Graph.mapMaterializedValue(srcGraph, notUsed -> new Pair<>(notUsed, notUsed));
-    Sink<Object, CompletionStage<Done>> snk = Sink.ignore();
-    Pair<NotUsed, NotUsed> pair = Source.fromGraph(mappedMatValueSrcGraph).to(snk).run(system);
-    assertEquals(pair, new Pair<>(NotUsed.getInstance(), NotUsed.getInstance()));
+    // FIXME it looks like the Graph.mapMaterializedValue static forwarder is no longer
+    // generated on Scala 3
+    //    Graph<SourceShape<Object>, Pair<NotUsed, NotUsed>> mappedMatValueSrcGraph =
+    //        Graph.mapMaterializedValue(srcGraph, notUsed -> new Pair<>(notUsed, notUsed));
+    //    Sink<Object, CompletionStage<Done>> snk = Sink.ignore();
+    //    Pair<NotUsed, NotUsed> pair =
+    // Source.fromGraph(mappedMatValueSrcGraph).to(snk).run(system);
+    //    assertEquals(pair, new Pair<>(NotUsed.getInstance(), NotUsed.getInstance()));
```

Comment:
Not sure how to move on with this. Raise a separate issue and unblock this PR? Hold this PR until we find a solution?

Sounds like a blocker.

Reply:
I'm not sure why it'd be a blocker: having a Scala 3 artifact without the `Graph.mapMaterializedValue` javadsl API method seems like an improvement over having no Scala 3 artifact at all ;), and it's still there on the 2.12 artifacts.

I'll create a proper issue though.

Reply:
I saw you raised an issue and linked to it. Works for me.